integration-tests/bats/sql-client.bats

```diff
@@ -58,14 +58,16 @@ teardown() {
     create_test_table
     run show_tables
     [ "$status" -eq 0 ]
+    echo $output
```

Comment:
(minor) you can pull out this debugging line. btw... I just recently learned (from Neil) about the `--verbose-run` flag when running bats tests locally – it automatically outputs `$output` for any `run` call. Pretty handy for debugging bats failures (locally at least), although I sometimes still use `echo $output` calls, especially if I can't repro something locally and need to see the output in the CI job. 

```suggestion
```